creme/billing/migrations/0002_productline_order_serviceline_order.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.10 on 2023-02-25 05:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('billing', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='productline',
+            name='order',
+            field=models.PositiveIntegerField(default=0, editable=False),
+        ),
+        migrations.AddField(
+            model_name='serviceline',
+            name='order',
+            field=models.PositiveIntegerField(default=0, editable=False),
+        ),
+    ]
```

Comment:
Rename this file `0028_v2_5__lines_order.py` to follow our naming convention (see migrations/old/).
Beware I have some billing migrations in a nearly merged PR too, so one of us will have to rename his files haha.